frontend/src/Components/forms/AddUserForm.tsx

```diff
@@ -73,7 +78,7 @@ export default function AddUserForm({
         reset();
         onSuccess(
             (response.data as NewUserResponse).temp_password,
-            'User created successfully with temporary password',
+            `${userRole} created successfully with temporary password`,
```

Comment:
Comment to title-case this line (e.g., "Admin created successfully with temporary password" or "Student created successfully with temporary password")